AK/Endian.h

```diff
@@ -134,6 +135,22 @@ class [[gnu::packed]] BigEndian {
 template<typename T>
 using NetworkOrdered = BigEndian<T>;
 
+template<typename T>
+requires(HasFormatter<T>) struct Formatter<LittleEndian<T>> : Formatter<FormatString> {
```

Comment:
Is there an implicit operator T for {Big,Little}Endian<T>?
If so, you can just inherit from `Formatter<T>` instead and not override anything